GLAA.Web/Startup.cs

```diff
@@ -125,6 +125,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IFormDefinition, LicenceApplicationFormDefinition>();
             services.AddTransient<IFieldConfiguration, FieldConfiguration>();
             services.AddTransient<IConstantService, ConstantService>();
+            services.AddTransient<IReferenceDataProvider, ReferenceDataProvider>();
```

Comment:
If we're not changing country and county at runtime, can this be a singleton and get the data on instantiation? just thinking we only need to get the data once for the app lifetime really don't we?

Reply:
Good spot!